### ISSUE TITLE
Audio sync issues with Shield Pro

### ISSUE THREAD

**Issue by pmindemann11** [Stremio/stremio-bugs#937]
Like others have reported, I'm having audio sync problems on my Nvidia Shield Pro. Here's the details...

**Describe the bug**
Audio does not sync correctly with streamed content (movies and recorded TV, but live TV appears ok). 

**To Reproduce**
I'm using an Nvidia Shield running version 11, Android TV software 9.1.1 and the Stremio app downloaded directly of the Stremio website, which is Android TV 64-bit ARM app version 1.6.12. The bug is reproduceable with any movie or TV show, but only on the Nvidia shield. There is approximately 0.25 sec of disparity through both Exo and VLC players.

**Expected behavior**
I am able to "fix" the issue by adding a 0.25 delay with VLC playback, but this has to be set on every show, every time. If there was a univeral way to apply the setting it wouldn't be as much of an issue, but VLC doesn't handle things like subtitles as well as Exo, either.

**Additional context**
- Tried running Stremio with and without VPN (I know it's not required, but I use Proton VPN). It made no difference.
- Checked the Shield's audio out settings, and most everything is set to auto (with no problems from other apps). I have a basic, 2.1-channel setup.
- Turned on hardware acceleration and made sure tunneled playback was off.
- Have switched from ExoPlayer to VLC and it does the same thing (though with a 0.25 delay added to VLC, it lines up). 
- The sync issue only appears on movie/recorded TV streams, not on live TV.
- Tested Stremio on other devices, and it works and syncs fine on plain Google TV with Chromecast dongles (for example). It just seems to be the Shield that has audio sync issues.

Thank you for the help!

**Comment by CLHdevOps** [comment]
Im experiencing the same. Did you find a fix?

**Comment by pmindemann11** [comment]
Yeah, I had to download and switch to JustPlayer for playback. Now the sync is spot-on, EVERY time!

**Comment by TRtomasz** [comment]
Try play with the following settings inside stremio: `Tunneled Playback` and `Hardware accelerated decoding` try differnt combinations on/off should help.
